diff --git a/mk/install.sh.template b/mk/install.sh.template
index 990fc00..6c147e1 100755
--- a/mk/install.sh.template
+++ b/mk/install.sh.template
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -e # Exit with nonzero exit code if anything fails
 
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 INSTALL_DIR={{INSTALL_DIR}}
 
 # copy libs
@@ -10,14 +11,18 @@ cp libCommon++.a libPacket++.a libPcap++.a $INSTALL_DIR/lib
 # copy header files
 mkdir -p $INSTALL_DIR/include
 mkdir -p $INSTALL_DIR/include/pcapplusplus
-cp header/* $INSTALL_DIR/include/pcapplusplus
+if [ -d header ]; then
+    cp header/* $INSTALL_DIR/include/pcapplusplus
+fi
 
 # copy examples
 mkdir -p $INSTALL_DIR/bin
-cp examples/* $INSTALL_DIR/bin
+if [ -d examples ]; then
+    cp examples/* $INSTALL_DIR/bin
+fi
 
 # create template makefile 
-cp mk/PcapPlusPlus.mk PcapPlusPlus.mk
+cp $SCRIPT_DIR/PcapPlusPlus.mk PcapPlusPlus.mk
 sed -i.bak '/PCAPPLUSPLUS_HOME :=/d' PcapPlusPlus.mk && rm PcapPlusPlus.mk.bak 
 sed -i.bak '/# libs dir/d' PcapPlusPlus.mk && rm PcapPlusPlus.mk.bak
 sed -i.bak '/PCAPPP_LIBS_DIR :=/d' PcapPlusPlus.mk && rm PcapPlusPlus.mk.bak
