diff --git a/Pcap++/Makefile b/Pcap++/Makefile
index 96eb409..30905e2 100755
--- a/Pcap++/Makefile
+++ b/Pcap++/Makefile
@@ -46,14 +46,11 @@ INCLUDES := -I"./src" \
 			-I"$(PACKETPP_HOME)/header" \
 			-I"$(LIGHT_PCAPNG_HOME)/include" \
 			-I"$(PCAPPLUSPLUS_HOME)/3rdParty/EndianPortable/include"
-			
+
 ifdef WIN32
 INCLUDES += -I$(MINGW_HOME)/include/ddk \
 			-I$(PCAP_SDK_HOME)/Include
 endif
-ifdef LINUX
-INCLUDES += -I/usr/include/netinet
-endif
 ifdef PF_RING_HOME
 INCLUDES += -I$(PF_RING_HOME)/userland/lib -I$(PF_RING_HOME)/kernel
 endif
@@ -78,7 +75,7 @@ endif
 ifdef USE_DPDK
 PCAPPP_BUILD_FLAGS += -msse -msse2 -msse3 -mssse3
 endif
-	
+
 Obj/%.o: src/%.cpp
 	@echo Building file: $<
 	@$(CXX) $(DEPS) $(INCLUDES) $(PCAPPP_BUILD_FLAGS) -c -fmessage-length=0 -MMD -MP -MF"$(@:Obj/%.o=Obj/%.d)" -MT"$(@:Obj/%.o=Obj/%.d)" -o "$@" "$<"
